components/studio/bin/hab-studio.sh

```diff
@@ -1160,6 +1097,33 @@ set_libexec_path() {
   return 0
 }
 
+defaults_path() {
+    echo "$($bb dirname "${libexec_path}")/defaults"
+}
+
```

Comment:
I realize this is _super_ nitpicky, but I'm curious why this is a function instead of just a variable, e.g.

`defaults_path="$($bb dirname "${libexec_path}")/defaults"`

Reply:
Hrmm... good point. In a previous incarnation of this PR there were a lot more call sites, but with the subsequent refactoring, it's only used in one place, so there's no reason it couldn't just be an internal variable.

Will change!

Reply:
Fixed in https://github.com/habitat-sh/habitat/pull/6319/commits/ae0e6a8852ee90b5860e00f09e528316fcb82ebf